src/middleware/jsonld.ts

```diff
@@ -0,0 +1,20 @@
+import jsonld from 'jsonld';
+import { Context as JsonLdContext } from 'jsonld/jsonld-spec';
```

Comment:
This is a TypeScript-only file, eslint can't find it so must be configured wrong.

Reply:
https://github.com/libero/article-store/pull/17